### DISCUSSION TITLE
1KB empty subtitle files after transcribing

### DISCUSSION THREAD

**Discussion by Fetterbr61** [SubtitleEdit/subtitleedit#8525] [General]
I am getting many empty subtitle files after trascribtion.  I use Purfviews batch with L 3 and translate to english on.  I testet a little and got the below log, can someone see what could be the error please. Thanks

-----------------------------------------------------------------------------
Date: 06/13/2024 07:48:15
SE: 4.0.6.0 - Microsoft Windows NT 10.0.19045.0 - 64-bit
Message: C:\Users\crist\AppData\Roaming\Subtitle Edit\Whisper\Purfview-Whisper-Faster\whisper-faster.exe --language ja --model "large-v3" --task translate --standard_asia "D:\VLC Media folder\Javluv New Movies Latest edition\INU-052 (2013)\INU-052 (2013).wmv"

-----------------------------------------------------------------------------
Date: 06/13/2024 07:48:41
SE: 4.0.6.0 - Microsoft Windows NT 10.0.19045.0 - 64-bit
Message: Calling whisper (Purfview's Faster-Whisper) with : C:\Users\crist\AppData\Roaming\Subtitle Edit\Whisper\Purfview-Whisper-Faster\whisper-faster.exe --language ja --model "large-v3" --task translate --standard_asia "D:\VLC Media folder\Javluv New Movies Latest edition\INU-052 (2013)\INU-052 (2013).wmv"
Warning: Word-level timestamps on translations may not be reliable.

Warning: 'large-v3' model may produce inferior results, try 'large-v2'!

Standalone Faster-Whisper r192.3 running on: CUDA
Starting work on: D:\VLC Media folder\Javluv New Movies Latest edition\INU-052 (2013)\INU-052 (2013).wmv
overflow (1870 > 1864) in spectral RLE, ignoring

[1288] Failed to execute script '__main__' due to unhandled exception!

av.error.PermissionError: [Errno 1] Operation not permitted; last error log: [wmav2] overflow (1870 > 1864) in spectral RLE, ignoring

File "av\\error.pyx", line 336, in av.error.err_check

File "av\\codec\\context.pyx", line 416, in av.codec.context.CodecContext._send_packet_and_recv

File "av\\codec\\context.pyx", line 513, in av.codec.context.CodecContext.decode

File "av\\stream.pyx", line 168, in av.stream.Stream.decode

File "av\\packet.pyx", line 87, in av.packet.Packet.decode

File "av\\container\\input.pyx", line 212, in decode

File "faster_whisper\audio.py", line 80, in _ignore_invalid_frames

File "faster_whisper\audio.py", line 90, in _group_frames

File "faster_whisper\audio.py", line 103, in _resample_frames

File "faster_whisper\audio.py", line 52, in decode_audio

File "faster_whisper\transcribe.py", line 315, in transcribe

File "D:\whisper-fast\__main__.py", line 1476, in cli

File "D:\whisper-fast\__main__.py", line 1600, in <module>

Traceback (most recent call last):
Calling whisper Purfview's Faster-Whisper done in 00:00:26.0097232
Loading result from STDOUT

**Comment by niksedk** [comment]
I'm not sure what the problem is... but you try latest beta and see if it's any better?
https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.6/SubtitleEditBeta.zip

**Reply by Fetterbr61** [reply]
thanks, i have installed it and it seams to be working, not sure what did it.  I have a RTX 4070 TI with a lot of cudas, what is the best setup for this and the quickest, i can see the XXL and Faster Whisper is very slow on my rig, maybe i am just doing it wrong.

**Reply by niksedk** [reply]
Thx for testing :)
Apparently Purfview Faster Whisper cannot handle wmv files. @Purfview: Correct  ?
You can use a smaller model to make it go faster. 
`large-v2` is the recommended model as `large-v3` does not seem to work quite as well...
Also, you might get better translate result in other translation engines like DeepL or Google Translate.

**Reply by Purfview** [reply]
Looks like some ffmpeg error with audio decoding, try `--ff_dump`.

**Reply by Fetterbr61** [reply]
> Looks like some ffmpeg error with audio decoding, try `--ff_dump`.

How do you do a --ff dump

**Comment by Fetterbr61** [comment]
Hej og tak for dit svar. I have actually tryed and payed for the tranlation
in your program but i can't get it to work.

fre. 14. jun. 2024 20.40 skrev Nikolaj Olsson ***@***.***>:

**Comment by Fetterbr61** [comment]
Hi.
It seams after a little testing that a fresh restart of the computer makes the error go away. I can transcribe without errors. Its more likely that after some time of work with different things and then making the transcribe will faile with 1 kb files.  I am not sure if there could be something with the app and integration into the system, memory or other things. But as i said a restart fixes the problem for the time being.
